tests/func/test_checkout.py

```diff
@@ -134,9 +134,10 @@ def outs_info(self, stage):
         FileInfo = collections.namedtuple("FileInfo", "path inode")
 
         paths = [
-            path
+            os.path.join(root, f)
             for output in stage["outs"]
-            for path in walk_files(output["path"])
+            for root, _, files in os.walk(output["path"])
```

Comment:
Looks like a bug, why did you switch to `os.walk()` here?

Reply:
Because walk_files requires dvcignore now. And I assumed that in case of this tests using dvcignore would be unnecessary overhead. Do you think I should create DvcIgnoreFilter in this test and use walk_files?

Reply:
`self.dvc.dvcignore`